packages/gatsby/src/redux/run-sift.js

```diff
@@ -187,8 +393,39 @@ const runSiftOnNodes = (nodes, args, getNode) => {
   if (firstOnly) {
     return handleFirst(siftFilter, nodes)
   } else {
-    return handleMany(siftFilter, nodes, queryArgs.sort, resolvedFields)
+    return handleMany(siftFilter, nodes, resolvedFields)
```

Comment:
the parameter `resolvedFields` is not used:
```javascript
function handleMany(siftArgs, nodes) {
```

Reply:
Good catch :)